### ISSUE TITLE
 Multi-category tracking

### ISSUE THREAD

**Issue by Zhengzhiyang0000** [mikel-brostrom/yolov8_tracking#711]
### Search before asking

- [X] I have searched the Yolov5_StrongSORT_OSNet [issues](https://github.com/mikel-brostrom/Yolov5_StrongSORT_OSNet/issues) and found no similar bug report.

### Question

Hello, can you use your tracking to track multiple categories of targets? Or can it only be used for one type of tracking?

**Comment by Zhengzhiyang0000** [comment]
Hello, I can detect multiple categories when I run track.py, but when I save save-txt, why are all targets of the same category in the txt file, and how to display each category in the tracking result txt file?

**Comment by henriksod** [comment]
Hi! If you take a look at [this site](https://motchallenge.net/instructions/), the class of a track is not part of the MOT compliant results. --save-txt stores the results in a MOT-compliant format.

We could make it possible to export in a MOTS Challenge compliant format, then the class should be included. @mikel-brostrom what do you think? Yolov8 comes with segmentation now right?

**Comment by mikel-brostrom** [comment]
> We could make it possible to export in a MOTS Challenge compliant format, then the class should be included. @mikel-brostrom what do you think? Yolov8 comes with segmentation now right?

Yes. MOTS compliant results are however not saved at the moment. Could be added to the TODO list :smile:
